src/generators/go/Constants.ts

```diff
@@ -0,0 +1,32 @@
+export const RESERVED_GO_KEYWORDS = [
+  'break',
+  'case',
+  'chan',
+  'const',
+  'continue',
+  'default',
+  'defer',
+  'else',
+  'enum',
```

Comment:
`enum` is not a keyword in Go. See https://go.dev/ref/spec#Keywords.

Reply:
Noted 🙂